app/controllers/admin/users_controller.rb

```diff
@@ -1,8 +1,10 @@
-class Admin::UsersController < Admin::ApplicationController
-  def index
-    ransack_params = params[:q]
-    @q = User.search(ransack_params)
-    @users = @q.result(distinct: true).
-             paginate(page: params[:page], per_page: 20)
+module Admin
+  class UsersController < Admin::BaseController
```

Comment:
__[reek]__
* [InstanceVariableAssumption] assumes too much for instance variable @q [:ref](https://github.com/troessner/reek/blob/master/docs/Instance-Variable-Assumption.md)

<!-- Comment by [SideCI](https://sideci.com) -->